rfcs/0080-nixos-release-schedule

```diff
@@ -0,0 +1,69 @@
+---
+feature: nixos-release-schedule
+start-date: 2020-11-28
+author: Jonathan Ringer (@jonringer)
+co-authors: Frederik Rietdijk (@FRidh)
+related-issues:
+---
+
+# Summary
+[summary]: #summary
+
+NixOS is released twice a year; the current release schedule targets March
+and September for the release months. However, trying to provide a polished
+desktop-user experience is difficult and moving the target release months to
+May and November should minimize the amount of in-house work needed to
+cut a release.
+
+# Motivation
+[motivation]: #motivation
+
+By changing our release dates to follow those of certain upstream projects
+we could reduce the amount of issues we encounter during stabilization.
+In particular, Gnome and KDE Plasma both have a release in October, which
+allows ample time for NixOS contributors to package and stabilize the
+desktop managers before the November release. For the May release, Gnome will
+have a similar stabilization period; while Plasma, which has a four month
+release cadence, will have a slightly older but still supported non-LTS
+release. Using LTS releases is not feasible as there is no guarantee that
+they will be compatible with the version of systemd during branch-off.
+
+The current March and September release months may be the worst months for
+Gnome and Plasma, as the release available during stabilization would be
+End-of-Life a few weeks after the target release date. Also, this period marks
+the greatest potential dependency "drift" from development on unstable. For example,
+many of the 20.09 release blockers were related to issues with systemd-246 and plasma-5.18,
+however, plasma-5.19 and the yet-released 5.20 both supported systemd-246 with
+zero additional effort.
+
+By choosing branch-off points in master where the major desktop manager use
+cases should be well stabilized will allow for release stabilization to
+focus on failing builds and minor issues.
+
+## Core changes
+
+The 21.03 release will be delayed until 21.05, and the current 20.09 release
+will be supported until June 2021. After the initial 21.05 release, all subsequent
```

Comment:
Specifying the exact time 20.09 is supported until is a change of policy. The current policy is to keep the previous release supported for 1 month after the next release.

Reply:
This is in regards to https://nixos.org/manual/nixos/unstable/release-notes.html#sec-release-20.09 in which we say April of 2021